security/pkg/nodeagent/secretfetcher/secretfetcher.go

```diff
@@ -104,6 +106,21 @@ type SecretFetcher struct {
 	FallbackSecretName string
 }
 
+// CertAndKey wraps a certificate, key, root certificate and certificate revocation list.  It also stores a
+// boolean to indicate if the CertAndKey is considered valid.
+type CertAndKey struct {
```

Comment:
is there a preferred name for this struct and function below?  Left as `CertAndKey` to match original function name, but if something else like `TLSContext` would be better let me know.